src/popup/components/TransactionItem.vue

```diff
@@ -168,6 +172,10 @@ export default defineComponent({
       ));
     });
 
+    onBeforeMount(() => {
+      setTransaction(props.transaction);
+    });
```

Comment:
This bothers me little bit. If we are assuming that the `transaction` prop is ready before mounting why not just use the `setTransaction` right at the beginning of the `setup` hook?
Or maybe we need to ensure that the transaction is not empty by watching the `transaction` prop?